modules/_labs/tabs/react/lib/cursor/hooks.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {mergeProps} from '@workday/canvas-kit-react-common';
+
+import {CursorModel, getItem} from './useCursorModel';
+
+export const orientationKeyMap = {
+  horizontal: {
+    ArrowLeft: 'previous',
+    Left: 'previous',
+    ArrowRight: 'next',
+    Right: 'next',
+    Home: 'first',
+    End: 'last',
+  },
+  vertical: {
+    ArrowUp: 'previous',
+    Up: 'previous',
+    ArrowDown: 'next',
+    Down: 'next',
+    Home: 'first',
+    End: 'last',
+  },
+} as const;
+
+/**
+ * Handles the roving focus behavior of a Cursor model. It does not handle the tabIndex. Use
+ * `useRovingTabIndex` for that functionality.
+ * @see https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex
+ */
+export const useRovingFocus = ({state, events}: CursorModel, elemProps = {}) => {
```

Comment:
Behavior hooks are tied to models, so this will go where the `CursorModel` goes in the future. Our packages are currently tied to components and not to more abstract concepts (except common). Perhaps `List` and `Cursor` stay abstract concepts and common makes sense.